internal/export/batcher.go

```diff
@@ -94,11 +94,11 @@ func (s *Server) handleCreateBatches() http.Handler {
 			switch {
 			case errors.Is(err, context.DeadlineExceeded):
 				logger.Infow("batch creation timed out")
-				w.WriteHeader(200)
+				w.WriteHeader(http.StatusOK)
```

Comment:
@mikehelmick these don't seem like the right statuses for these?

Reply:
this one should probably be 500